pages/blog/2023-summary.md

```diff
@@ -0,0 +1,164 @@
+
+Let's dive into the data:
+
+#### LinkedIn
+
+For our LinkedIn page, we are currently sitting at **2988** overall followers, and during 2023, we gained a total of **849** followers since 2022.
+
+<Figure
+  src="/img/posts/2023-summary/linkedin.webp"
+/>
+
+We also don't use any premium or marketing services; everything you see here is 100% organic, and even though LinkedIn still has much to offer users, it is way better than the X/Twitter app, in my opinion.
+
+<Figure
+  src="/img/posts/2023-summary/referred-by-LinkedIn.webp"
+/>
+
+#### X/Twitter
+
+I don't have much to say regarding Twitter, but on the bright side, we gained almost 1K followers since 2022, and we currently have 4.4k overall followers. When it comes to website referrals via Twitter, we have seen a decline compared to last year, 2022
+
+<Figure
+  src="/img/posts/2023-summary/referred-by-Twitter.webp"
+/> 
+
+Although the decline is shallow, I'm not extremely worried. Most top tweets for this year were appreciation tweets balanced up with technical tweets. Additionally, we don't use any paid services, and having any additional investment in this platform is not worth it, at least for now.
+
+#### YouTube
+
+The AsyncAPI YouTube channel performed way better than expected, especially as we moved from online to in-person conferences. We currently have over 364 people subscribed to our channel, increasing total subscribers to over 1,46k. Our live streams and conference recordings bring in a lot of views, and I'm optimistic about 2024 as well.
+
+<Figure
+  src="/img/posts/2023-summary/youtube.webp"
+/> 
+
+#### Newsletter
+On the 5th of April, 2023, we sent out our first AsyncAPI Community Updates Newsletter. The main aim was to summarize monthly activities and initiatives in the community straight to your inbox.
+Over the past nine months, the December Edition was the most popular. This was due to the release of the AsyncAPI v3, which was one of our major accomplishments in 2023.
+
+<Figure
+  src="/img/posts/2023-summary/total-opens.webp"
+/> 
+
+And, of course, I'm not exempt from unsubscribers. Although the number might seem minimal to some, it still stings as a writer.
+
+<Figure
+  src="/img/posts/2023-summary/unsubscribed.webp"
+/> 
+
+Looking forward to sending the 2024 Editions!!🎉🎉
+
+### Google Analytics
+
+The pageviews on the AsyncAPI website increased significantly over the past year. Our major contributor to these metrics comes from docs, and we expect to see more in the coming year as people want to know more about the AsyncAPI v3.
+
+<Figure
+  src="/img/posts/2023-summary/google-analytics.webp"
+/> 
+
+We also lost 21 days' worth of data during October due to a PR update that affected the analytics workflow, but we are doing well.
+
+### Google Search Console
+
+Google Search Console is one of my favorite tools as it shows how much impact and interest we are gaining. Compared to 2022, we can see a surge in impressions and, slowly but surely, in clicks as well. 
```

Comment:
```suggestion
Google Search Console is one of my favorite tools as it shows how much impact and interest we are gaining. Compared to 2022, we can see a surge in impressions and clicks. 
```

It would be better to write more data points in the actual blog post, vs only showing images with data.